service_container/factories.rst

```diff
@@ -38,15 +38,63 @@ configure the service container to use the
         services:
             app.newsletter_manager:
                 class:   AppBundle\Email\NewsletterManager
-                # call a static method
-                factory: ['AppBundle\Email\NewsletterManager', create]
+                # call the static method
+                factory: ['AppBundle\Email\NewsletterManagerStaticFactory', createNewsletterManager]
 
+    .. code-block:: xml
+
```

Comment:
I've done this. Am however wondering whether it's not `app/config/services.yml` and `app/config/services.php` to be exactly?